rm-datasource/src/main/java/io/seata/rm/datasource/sql/SQLVisitorFactory.java

```diff
@@ -66,7 +66,19 @@ public static SQLRecognizer get(String sql, String dbType) {
                     recognizer = new MySQLSelectForUpdateRecognizer(sql, ast);
                 }
             }
-        } else {
+        }  else if (JdbcConstants.ORACLE.equalsIgnoreCase(dbType)) {
```

Comment:
Not sure if `equalsIgnoreCase` is really required here. Otherwise a switch statement would be a better solution.